Program.cs

```diff
@@ -123,40 +156,27 @@ public static void Main()
             if (!gameOver && currentFrame - lastFrameMoved >= secondsToMove)
             {
                 lastFrameMoved = currentFrame;
-                snake.Move(gridRows, gridCols);
-                gameOver = snake.Overlaps(snake.snakeRow, snake.snakeCol, score, skipHead: true);
+                snake.Move(grid);
+                gameOver = snake.Overlaps(snake.Head, score, skipHead: true);
             }
 
-            // Debug code for printnig arrays 
-            // foreach (var item in snake.prevLocRow)
-            // {
-            //     Console.Write(item.ToString(), ", ");
-            // }
-            // Console.WriteLine();
-
-            // foreach (var item in snake.prevLocCol)
-            // {
-            //     Console.Write(item.ToString(), ", ");
-            // }
-            // Console.WriteLine();
-
-            if (snake.Overlaps(apple.row, apple.col, score))
+            if (snake.Overlaps(apple.appleCords, score))
```

Comment:
I think it's odd that you have `apple.appleCords` this seems like it's duplicating information and also coords have 2 o, I think it should be `apple.Coords`. Just remember **if you name your variables well and our fields well your code should explain itself**. This will come with time, but clearly here you are doing extra duplicate visual information. 

it's ok to do this btw. the code works but getting a feel for code cleanliness and style is also good.